### PR TITLE
no more vars in Web/API (5)

### PR DESCRIPTION
**Pull request by lionralfs**
#### Summary

Replaces `var` in Web/API with `const` where applicable and `let` otherwise.
Mostly automated via my ESLint setup, using the rules `no-var` and `prefer-const`

#### Motivation

* See #16662
* See https://github.com/mdn/content/discussions/10389
* See https://github.com/orgs/mdn/discussions/143

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/screen/unlockorientation/index.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Returns `true` if the orientation was successfully unlocked or
 ## Examples
 
 ```js
-var unlockOrientation = screen.unlockOrientation || screen.mozUnlockOrientation || screen.msUnlockOrientation || (screen.orientation && screen.orientation.unlock);
+const unlockOrientation = screen.unlockOrientation || screen.mozUnlockOrientation || screen.msUnlockOrientation || (screen.orientation && screen.orientation.unlock);
```
Comment (teoli2003) [inline files/en-us/web/api/screen/unlockorientation/index.md:43]:
```suggestion
const unlockOrientation = screen.mozUnlockOrientation || screen.msUnlockOrientation;
```

File: files/en-us/web/api/selection/collapse/index.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ None ({{jsxref("undefined")}}).
 
 ```js
 /* Place the caret at the beginning of an HTML document's body. */
-var body = document.getElementsByTagName("body")[0];
+const body = document.getElementsByTagName("body")[0];
```
Comment (teoli2003) [inline files/en-us/web/api/selection/collapse/index.md:45]:
```suggestion
const body = document.querySelector("body");
```

File: files/en-us/web/api/selection/setbaseandextent/index.md [hunk 3]
```diff
@@ -92,23 +92,23 @@ selection into the output paragraph at the very bottom of the HTML.
...
+let selection;
 
 button.onclick = function() {
   try {
     selection = document.getSelection();
     selection.setBaseAndExtent(one, aOffset.value, two, fOffset.value);
-    var text = selection.toString();
+    const text = selection.toString();
     output.textContent = text;
   } catch(e) {
```
Comment (teoli2003) [inline files/en-us/web/api/selection/setbaseandextent/index.md:113]:
```suggestion
  } catch (e) {
```

File: files/en-us/web/api/textdecoder/textdecoder/index.md [hunk 4]
```diff
@@ -43,10 +43,10 @@ new TextDecoder(utfLabel, options)
 ## Examples
 
 ```js
-var textDecoder1 = new TextDecoder("iso-8859-2");
-var textDecoder2 = new TextDecoder();
-var textDecoder3 = new TextDecoder("csiso2022kr", {fatal: true}); // Allows TypeError exception to be thrown.
-var textDecoder4 = new TextDecoder("iso-2022-cn"); // Throw a RangeError exception.
+const textDecoder1 = new TextDecoder("iso-8859-2");
+const textDecoder2 = new TextDecoder();
+const textDecoder3 = new TextDecoder("csiso2022kr", {fatal: true}); // Allows TypeError exception to be thrown.
```
Comment (teoli2003) [inline files/en-us/web/api/textdecoder/textdecoder/index.md:48]:
```suggestion
const textDecoder3 = new TextDecoder("csiso2022kr", { fatal: true }); // Allows TypeError exception to be thrown.
```

File: files/en-us/web/api/timeranges/length/index.md [hunk 5]
```diff
@@ -26,11 +26,11 @@ A number.
 Given a video element with the ID "myVideo":
 
 ```js
-var v = document.GetElementById("myVideo");
+const v = document.GetElementById("myVideo");
```
Comment (teoli2003) [inline files/en-us/web/api/timeranges/length/index.md:29]:
```suggestion
const v = document.getElementById("myVideo");
```

File: files/en-us/web/api/urlsearchparams/entries/index.md [hunk 6]
```diff
@@ -38,10 +38,10 @@ Returns an {{jsxref("Iteration_protocols","iterator")}}.
 
 ```js
 // Create a test URLSearchParams object
-var searchParams = new URLSearchParams("key1=value1&key2=value2");
+const searchParams = new URLSearchParams("key1=value1&key2=value2");
 
 // Display the key/value pairs
-for(var pair of searchParams.entries()) {
+for(const pair of searchParams.entries()) {
    console.log(`${pair[0]}, ${pair[1]}`);
```
Comment (teoli2003) [inline files/en-us/web/api/urlsearchparams/entries/index.md:45]:
```suggestion
for(const [key, value] of searchParams.entries()) {
   console.log(`${key}, ${value}`);
```

File: files/en-us/web/api/urlsearchparams/foreach/index.md [hunk 7]
```diff
@@ -49,7 +49,7 @@ None ({{jsxref("undefined")}}).
 
 ```js
 // Create a test URLSearchParams object
-var searchParams = new URLSearchParams("key1=value1&key2=value2");
+const searchParams = new URLSearchParams("key1=value1&key2=value2");
 
 // Log the values
 searchParams.forEach(function(value, key) {
```
Comment (teoli2003) [inline files/en-us/web/api/urlsearchparams/foreach/index.md:55]:
```suggestion
searchParams.forEach((value, key) => {
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/screen/unlockorientation/index.md (hunk 1 above)
Comment (lionralfs) [reply files/en-us/web/api/screen/unlockorientation/index.md:43]: I'm not sure about this one. Shouldn't the non-vendor-specific methods stay?

Comment (teoli2003) [reply files/en-us/web/api/screen/unlockorientation/index.md:43]: Let's keep it.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**